Added inspection workflow-templates

CermakM opened the pull request:
Signed-off-by: Marek Cermak <macermak@redhat.com>

renamed:    examples/templates/template-collect-outputs.yaml -> templates/collect-outputs-template.yaml
renamed:    examples/templates/template-inspection-build.yaml -> templates/inspection-build-template.yaml
renamed:    examples/templates/template-inspection-run.yaml -> templates/inspection-run-template.yaml

Commit:
Added inspection workflow-templates to the root dir

Signed-off-by: Marek Cermak <macermak@redhat.com>

renamed:    examples/templates/template-collect-outputs.yaml -> templates/collect-outputs-template.yaml
renamed:    examples/templates/template-inspection-build.yaml -> templates/inspection-build-template.yaml
renamed:    examples/templates/template-inspection-run.yaml -> templates/inspection-run-template.yaml

ghost commented:
Build succeeded.

- [thoth-coala ](https://managesf.thoth-station.ninja/logs/14/14/e19c7aaee01517804cdd24772902ad365814ebe5/check/thoth-coala/4ef3289/) : SUCCESS in 44s

ghost commented:
Build succeeded.

- [thoth-coala ](https://managesf.thoth-station.ninja/logs/14/14/7773c61bcf5256039a7173cbf6d07266c09923e9/check/thoth-coala/1bdf1ff/) : SUCCESS in 40s

goern commented:
@CermakM how are the images built? where is the src?

CermakM commented:
@goern Currently https://github.com/CermakM/kubectl-image/blob/master/Dockerfile

Commit:
Fixed and specialized inspection output collection

Signed-off-by: Marek Cermak <macermak@redhat.com>

deleted:    collect-outputs-template.yaml
new file:   inspection-results-template.yaml

Commit:
Do not archive the artifacts

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   inspection-results-template.yaml

ghost commented:
Build succeeded.

- [thoth-coala ](https://managesf.thoth-station.ninja/logs/14/14/5b2df0b357515c334f01aafeda749249452392fd/check/thoth-coala/640b090/) : SUCCESS in 40s

ghost commented:
Build succeeded (gate pipeline).

- [thoth-coala ](https://managesf.thoth-station.ninja/logs/14/14/5b2df0b357515c334f01aafeda749249452392fd/gate/thoth-coala/cc76396/) : SUCCESS in 38s
